src/main/java/com/itacademy/cms/model/Tags.java

```diff
@@ -0,0 +1,33 @@
+package com.itacademy.cms.model;
+
+
+import lombok.*;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Builder
+@Table(name = "tags")
+public class Tags {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private int id;
+
+    @NotNull
+    @Column(name = "name")
+    private String name;
+
+    @ManyToMany(cascade = CascadeType.PERSIST)
+    @JoinTable(
+            name = "course_tags"
+            , joinColumns = @JoinColumn(name = "tags")
+            , inverseJoinColumns = @JoinColumn(name = "course"))
+    private List<Courses> courses;
```

Comment:
The name of our table is 'courses', and then columns to join are 'tag_id' and 'course_id'. Also the name of the class id Course in singular. Change it please